app/src/main/res/values/strings.xml

```diff
@@ -20,4 +20,5 @@
     <string name="title_visibility">Visibility</string>
     <string name="title_ozone">Ozone</string>
     <string name="title_o">o</string>
+    <string name="title_please_wait">Please wait...</string>
```

Comment:
WARNING: [TypographyEllipsis] Replace "..." with ellipsis character (…, &#8230;) ?
 You can replace the string "..." with a dedicated ellipsis character, ellipsis character (…, &#8230;). This can help make the text more readable.